test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -95,6 +95,9 @@ type ClusterUpgradeWithRuntimeSDKSpecInput struct {
 	ExtensionServiceNamespace string
 	// ExtensionServiceName is the name of the service to configure in the test-namespace scoped ExtensionConfig.
 	ExtensionServiceName string
+
+	// ClassNamespace is an optional class namespace reference, configuring cross-namespace cluster class reference
```

Comment:
the godoc sounds a bit strange and also the field name :)

Is this more like: `DeployClusterClassInSeparateNamespace` ?

Hm just noticed that it is a *string in the quickstart test. Do we care about the specific namespace used or is a boolean flag enough?
(would be good if both tests have ~ the same input parameter)

Reply:
I started with bool, but then realized that the NS name generation has a seed, `specName` in this case. And these names may overlap, so I left the value being specified on demand. May be also clear when collecting logs, where the resources are located (not just 2 randomly generated names as it would otherwise be)